.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
This file should not be commited. Please add it to the .gitignore. I think we should simply add .idea and .idea/** to .gitignore, so that no config files from this directory are commited.

Reply:
these are pycharm specific files that are not user-specific. Mostly they are kept in the repo. But since no one uses pycharm I removed the whole .idea folder.